src/account_manager/operations/participation.rs

```diff
@@ -62,6 +62,19 @@ impl AccountManager {
             .collect())
     }
 
+    pub async fn get_participation_events_from_client(
+        &self,
+        event_type: Option<ParticipationEventType>,
+    ) -> crate::Result<Vec<EventId>> {
+        let accounts = self.accounts.read().await;
+        let client = match accounts.first() {
+            Some(account) => account.client.clone(),
+            None => self.client_options.read().await.clone().finish()?,
+        };
+        let events = client.events(event_type).await?;
+        Ok(events.event_ids)
```

Comment:
```suggestion

        Ok(events.event_ids)
```

Reply:
Why is the function returning `event_ids` and not `events`?

Reply:
There are a few reasons:
1. The node returns just the events' IDs, not the corresponding event information / payload.
2. The events will have to be "registered" anyway, which requires an ID and a node. This function, `register_participation_events` returns `Result<Event>` anyway, so Firefly will have the data eventually.

Alternatively, we could expose an additional method e.g. `register_participation_events_of_node` that would return `Result<Vec<Event>>`, but since this would only be particularly useful for the official events on official nodes, it is alright IMO if Firefly just handles these extra steps.